sources/osgAnimation/BasicAnimationManager.js

```diff
@@ -345,13 +328,54 @@ define( [
         },
 
 
+        _registerAnimations: function () {
+
+            if ( !this._targets.length ) return;
+
+            for ( var i = 0, l = this._animationsToRegister.length; i < l; i++ ) {
+                var instanceAnimation = this._animationsToRegister[ i ];
+                this._registerInstanceAnimation( instanceAnimation );
+            }
+
+            this._animationsToRegister.length = 0;
+            this._dirty = false;
+        },
+
+        // Register animation
+        //
+        // Register animation list all target from channel in the animations and associate
+        // target found in the scenegraph. If no target are registered animation cant be
+        // registered. In this case animation will be pending and resolved after a visitor
+        // extract target.
+        _registerInstanceAnimation: function ( instanceAnimation ) {
+
+            var instanceChannels = instanceAnimation.channels;
+            for ( var i = 0, l = instanceChannels.length; i < l; i++ ) {
+                var instanceChannel = instanceChannels[ i ];
+                var targetName = instanceChannel.channel.target;
+                var name = instanceChannel.channel.name;
+                var uniqueTargetName = targetName + '.' + name;
+
+                // disply a warning if animation has a channel but not target found in the
+                // scene graph. We could probably optimize and removes those channels, but
+                // it must be a user decision in case the user plugin different scene
+                // graph together and target would appear later in the scenegraph
+                if ( !this._targetMap[ uniqueTargetName ] ) {
+                    Notify.warn( 'registerInstanceAnimation did not find targetName ('+ uniqueTargetName + ') in the scene graph' );
```

Comment:
k. 
( maybe add that those solution to the comment if user get the warning and want to fix it in his code)